warp10/src/main/java/io/warp10/continuum/gts/GTSEncoder.java

```diff
@@ -460,10 +472,10 @@ public synchronized int addValue(long timestamp, long location, long elevation,
         }
       } else {
         // Do nothing, implicitly we will encode location as raw GeoXPPoint
-        noDeltaMetaLocation = false;
       }
     } else {
       lastGeoXPPoint = GeoTimeSerie.NO_LOCATION;
+      validLastGeoXPPoint = false;
```

Comment:
`lastGeoXPPoint = GeoTimeSerie.NO_LOCATION;` is unnecessary since `validLastGeoXPPoint` is set to `false`.